ui.c

```diff
@@ -352,7 +352,7 @@ SCROLLABLE scroll_list = {
     .panel = {
         .type = PANEL_SCROLLABLE,
     },
-    .color = LIST_DARK,
+    .color = C_SCROLL,
```

Comment:
I've decided not to do scroll bar styling as part of this PR as it's likely to require a rewrite